diff --git a/node_modules/app-builder-lib/templates/nsis/portable.nsi b/node_modules/app-builder-lib/templates/nsis/portable.nsi
index 2e98dc2..e908549 100644
--- a/node_modules/app-builder-lib/templates/nsis/portable.nsi
+++ b/node_modules/app-builder-lib/templates/nsis/portable.nsi
@@ -8,7 +8,9 @@ AutoCloseWindow True
 RequestExecutionLevel ${REQUEST_EXECUTION_LEVEL}
 
 Function .onInit
-  SetSilent silent
+  !ifndef SPLASH_IMAGE
+    SetSilent silent
+  !endif
 
   !insertmacro check64BitAndSetRegView
 FunctionEnd
